wsgi_mod_rpaf.py

```diff
@@ -23,27 +22,30 @@ def text_to_wsgi(s):
         return s
 
 
-PROXY_IPS_DIRECTIVE = 'RPAFproxy_ips '
+def from_apache_config(f, directive='RPAFproxy_ips'):
```

Comment:
err, this should probably be changed to take a path rather than a file-like object